dotnet/ios/ios13/AddingTheSignInWithAppleFlowToYourApp/AddingTheSignInWithAppleFlowToYourApp/AppDelegate.cs

```diff
@@ -0,0 +1,29 @@
+namespace AddingTheSignInWithAppleFlowToYourApp;
+
+[Register ("AppDelegate")]
+public class AppDelegate : UIApplicationDelegate {
+	public override UIWindow? Window {
+		get;
+		set;
+	}
+
+	public override bool FinishedLaunching (UIApplication application, NSDictionary launchOptions)
+	{
+		// create a new window instance based on the screen size
+		Window = new UIWindow (UIScreen.MainScreen.Bounds);
+
+		// create a UIViewController with a single UILabel
+		var vc = new UIViewController ();
+		vc.View!.AddSubview (new UILabel (Window!.Frame) {
```

Comment:
```suggestion
		vc.View?.AddSubview (new UILabel (Window!.Frame) {
```

Rather than tell the compiler to ignore the warning which could result in a NRW use ? which will do nothing if null.

Reply:
Oh man, this isn't even the right project.. Rolf pointed out that I added a test project when I created this PR but I guess I also included this one...

Reply:
Okay, Just looked at all the files and it seems that there aren't any more accidentally added projects...